src/tiledb/cloud/client.py

```diff
@@ -63,6 +63,36 @@ def Ctx(config=None):
     return tiledb.Ctx(Config(config))
 
 
+class AWS(enum.Enum):
```

Comment:
I would call this AWSRegion or namespace it inside a Region class (so it shows up as `Region.AWS.whatever`)

Reply:
Fixed in 445c41d.